docs/consensus/scheduler.md

```diff
@@ -19,6 +19,19 @@ the [consensus service API documentation].
 
 ## Validator Committee
 
+To schedule the validator committee, the committee scheduler selects among
+nodes registered with the `RoleValidator` role.
```

Comment:
Link `registered` to `registry.md#register-node`. And link `RoleValidator` to https://pkg.go.dev/github.com/oasisprotocol/oasis-core/go/common/node?tab=doc#RoleValidator.

Reply:
oh good suggestions